src/components/AppComponents/AppHeaderBurgerMenuSections/AppHeaderBurgerMenuSections.tsx

```diff
@@ -22,8 +27,15 @@ const AppHeaderBurgerburgerMenuSections: FC<
   AppHeaderBurgerburgerMenuSectionsProps
 > = (props) => {
   const { burgerMenuSections } = props;
+  const { user } = useUser();
+
   return (
     <OakFlex $flexDirection="column" $gap="all-spacing-7">
+      {user && (
+        <OakSecondaryButton element="a" href="/api/auth/oidc-logout">
```

Comment:
This can be `/api/auth/logout`. I only added the `/oidc-logout` endpoint as a testing convenience. The difference between the two is subtle. `/logout` will clear the user's session in OWA, while `/oidc-logout` will clear the user's session in OWA and Auth0. It gives us a clean slate in Auth0 making it easier to test with different accounts.

Reply:
It only really becomes relevant when users can login to multiple applications.

This is a really helpful guide to the sessions https://auth0.com/docs/authenticate/login/logout

Probably helps to think of Auth0 as an auth platform and not login for a single app. 

Going forward we might want users to be able to login to Aila or to give 3rd parties access to APIs etc. and that can all go through Auth0. If the user has a session in Auth0 and the user has previously given the app access (I.e. logged into it before) then those logins can be silent with no prompt. If we trash that session on logout of any one app then the user will have to login again for any other app.

We can make that decision and alter the default behaviour of `/logout` to do a "full" logout if we want to. Or we could configure `/login` in OWA to always display the login prompt if we want to go that route.

That was why I added the `/oidc-logout` because when testing it's not helpful to be automatically logged in if you want to switch to another test account!

Reply:
thanks that's really helpful!